cmake/Modules/Packages/KIM.cmake

```diff
@@ -35,6 +35,8 @@ if(PKG_KIM)
   option(DOWNLOAD_KIM "Download KIM-API from OpenKIM instead of using an already installed one" ${DOWNLOAD_KIM_DEFAULT})
   if(DOWNLOAD_KIM)
     message(STATUS "KIM-API download requested - we will build our own")
+    # Workaround for cross compilation with MinGW, which messes up ${CMAKE_INSTALL_LIBDIR}
+    string(REPLACE ${CMAKE_INSTALL_PREFIX} "" _KIM_LIBDIR ${CMAKE_INSTALL_LIBDIR})
```

Comment:
So this basically takes the `/lib` port of `${CMAKE_INSTALL_LIBDIR}`, correct? The current comment is not very helpful....

Isn't `${CMAKE_INSTALL_LIBDIR}` just `lib` (or `lib64`) usually? And `${CMAKE_INSTALL_FULL_LIBDIR}` the full path?

Reply:
not when cross-compiling

Reply:
I guess a better comment would be something like: "When cross-compilation (e.g. MinGW) CMAKE_INSTALL_LIBDIR contains the absolute path, hence with strip the libdir portion".

Reply:
Also we might just want to do that once instead of here and in `LATTE.cmake`.